src/coreclr/jit/codegenwasm.cpp

```diff
@@ -80,13 +122,23 @@ void CodeGen::genCodeForTreeNode(GenTree* treeNode)
     switch (treeNode->OperGet())
     {
         case GT_ADD:
+        case GT_EQ:
+        case GT_NE:
+        case GT_LT:
+        case GT_LE:
+        case GT_GE:
+        case GT_GT:
             genCodeForBinary(treeNode->AsOp());
             break;
 
         case GT_LCL_VAR:
             genCodeForLclVar(treeNode->AsLclVar());
             break;
 
+        case GT_JTRUE:
+            // Do nothing; handled by end of block processing
+            break;
```

Comment:
To still echo the earlier discussion, I don't see a strong reason to make the WASM backend different from other backends here.

As-is, this is also missing `genConsumeOperands`.